components/director/internal/authenticator/claims.go

```diff
@@ -1,7 +1,7 @@
 package authenticator
 
 import (
-	"github.com/dgrijalva/jwt-go"
+	"github.com/form3tech-oss/jwt-go"
```

Comment:
this is a fork with 0 stars. Im not sure how reliable it is to switch to such forks. Would it be possible to get the security fix in https://github.com/dgrijalva/jwt-go instead ?

Reply:
currently, there is no fix in the https://github.com/dgrijalva/jwt-go and we do not know when will be provided such one. So we use **github.com/form3tech-oss/jwt-go** which is a fork of the same library in which the problem is fix